examples/static_graph_optimizations/mnist/train_mnist_custom_loop.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+"""Fully-connected neural network example using MNIST dataset
```

Comment:
Add `with static subgraph optimization` to make clear how this is different from the other one.